spec/features/home_spec.rb

```diff
@@ -143,4 +143,24 @@ def ie_alert_box_xpath
       "/html/body/div[@class='wrapper ']/comment()[contains(.,'ie-callout')]"
     end
   end
+  
```

Comment:
Would it be possible to remove the trailing whitespace on this line and a few more lines on this file?